src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -373,10 +373,12 @@ class ImageBaseViewer extends BaseViewer {
             if (!this.isPannable && this.isZoomable) {
                 // If the mouse up was not due to panning, and the image is zoomable, then zoom in.
                 this.zoom('in');
+                this.emitMetric(VIEWER_EVENT.imageZoomClick, 'in');
```

Comment:
It seems like we already have a `zoom` event emitted from the viewers -- could we enhance that to include an `interactionType` or something?

Reply:
`zoom`  event emitted from viewers doesn't get logged. Adding listener to viewer to specifically log `zoom` event in `Preview.js` adds a lot of complications to this PR. Instead, I just changed the event name to `zoom` for scalability and consistence.

Reply:
In what way does it add complexity?

Reply:
> In what way does it add complexity?

So now it's a two-lines change. If we use `emit('zoom')`, we need to add `zoom` event handler in `attachViewerListeners` in `Preview.js`, which sounds more complex than two lines.